tests/data/json-validation-cases.yaml

```diff
@@ -0,0 +1,3020 @@
+---
+- name: valid schema version
+  json: |
+    {
+      "schema_version": "1.1"
```

Comment:
use 1.3, because 1.2 will soon become valid :)

Reply:
There is no need to have everything in here, as many can be written in DSL format and imported.

That has the additional benefit of not needing to write many of the tests twice.

Only json that is not supported by the DSL needs to have extra tests